clientconfig_test.go

```diff
@@ -35,10 +71,18 @@ func testConfig(t *testing.T, data string) {
 			t.Errorf("domain is unexpected: %v", cc.Search[0])
 		}
 	}
+	if cc.Ndots != expectedNdots {
+		t.Errorf("Ndots not properly parsed: (Expected: %d / Was: %d)", expectedNdots, cc.Ndots)
+	}
 }
 
-func TestNameserver(t *testing.T)          { testConfig(t, normal) }
-func TestMissingFinalNewLine(t *testing.T) { testConfig(t, missingNewline) }
+func TestNameserver(t *testing.T)          { testConfig(t, normal, 1) }
+func TestMissingFinalNewLine(t *testing.T) { testConfig(t, missingNewline, 1) }
+func TestWithOutNdots(t *testing.T)        { testConfig(t, normalWithOutNdots, 1) }
```

Comment:
Instead of adding all these test function I would just add a single `TestNdots` tests, put all the new consts in a `map[string]int` and test the resulting ndots by just ranging over that map. Think even a resolv.conf with just `options ndots:N` is already valid, so this would be somewhat smaller.